packages/docusaurus/src/client/exports/ComponentCreator.tsx

```diff
@@ -43,6 +43,24 @@ export default function ComponentCreator(
       },
     });
   }
+  if (path === '/*') {
```

Comment:
This feels hacky, but I'm just not familiar enough with the ecosystem or Docusaurus code to make that call. So here goes :smile: